Headers/Foundation/NSOperation.h

```diff
@@ -37,7 +37,8 @@ extern "C" {
 #if OS_API_VERSION(MAC_OS_X_VERSION_10_6, GS_API_LATEST)
 #import <GNUstepBase/GSBlocks.h>
 DEFINE_BLOCK_TYPE_NO_ARGS(GSOperationCompletionBlock, void);
-#endif
+DEFINE_BLOCK_TYPE(GSBlockOperationBlock, void, void);
```

Comment:
Why didn't you use DEFINE_BLOCK_TYPE_NO_ARGS here?

Reply:
Didn't know of it's existence. :)